massbalancemachine/data_processing/Dataset.py

```diff
@@ -157,9 +154,13 @@ def create_glacier_grid(self,
         ds, glacier_indices, gdir = get_glacier_mask(self.data,
                                                      custom_working_dir)
         years_stake = self.data['YEAR'].unique()
-        years = range(years_stake.min(), years_stake.max() + 1)
+        years = range(1951, 2023)
```

Comment:
Why the fixed time range? Maybe comment on that if it is a quick fix for something that is not working?

Reply:
It's because we want the grid from the beginning of ERA5 to the end. The way it was done before it took the min and max of measurement years, but I actually also want to create glacier grids beyond that.